src/pages/WebMeetWaitingRoom.js

```diff
@@ -38,9 +38,9 @@ const WebMeetWaitingRoom = ({ patientDetails }) => {
   );
 
   const handleAppointmentCompleted = useCallback(
-    (data) => {
+    async (data) => {
       setAppointment({ ...appointment, eventStatusDesc: "Completed" });
-
+      await patientService.UpdateMessageStatus(hashKey);
```

Comment:
let's use `camelCase` for function names

Reply:
resolved